deployment/chainloop/templates/cas/gcp_secret_manager.secret.yaml

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Could we make sure these lines are not here? 

Ideally we should see here just the functionality that's been added, otherwise there is too much noise.